packages/grid/src/useGrid.ts

```diff
@@ -5,55 +5,27 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import { useState, HTMLProps, HTMLAttributes, KeyboardEventHandler } from 'react';
-import { composeEventHandlers, KEYS } from '@zendeskgarden/container-utilities';
```

Comment:
I'm surprised this import was working as expected before adding the dependency.. good thing for `tsconfig` path resolution.